Add image elements to Home TableView and Info TableView

NikolasGeorgaklis opened the pull request:
Both tableviews will display images; home will display user profile pictures as well as team logos and info will just display logos.

NikolasGeorgaklis committed:
Add image elements to Home TableView and Info TableView

Both tableviews will display images; home will display user profile pictures as well as team logos and info will just display logos.

vaguilar commented:
nitpitck: there are a couple lines added/removed, should leave these out. 

otherwise LGTM!

NikolasGeorgaklis committed:
Complete stats section

 I have completed the section in the middle tab bar item of the tab bar controller which displays first round matchups for the 2022 march madness tournament in addition to important odds(moneyline) and a label indicating which team is favorited.

vaguilar commented on NikoBracket/InfoViewController.m:
```diff
@@ -25,52 +29,77 @@ - (void)viewDidLoad {
+    
+    //parse 1st round matchups from json
+    NSError *error;
+    NSString *url_string = [NSString stringWithFormat: @"https://api.sportsdata.io/v3/cbb/scores/json/Tournament/2022?key=003d3b7e4d724cbeb785ca7d4aded519"];
```
It might also be good to put the API endpoint string as a constant at the top of the file or class, something like `const NSString kMatchUpsEndpoint = @"https://api.sportsdata.io/v3/cbb/scores/json/Tournament/2022?key=";`

NikolasGeorgaklis committed:
ignore Keys.plist and modify stats page

Put API key in Keys.plist and ignored Keys.plist
